unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -500,6 +500,14 @@ public static IntPtr unity_method_get_function_pointer([NativeCallbackType("Mono
         return IntPtr.Zero;
     }
 
+    [return: NativeCallbackType("gboolean")]
+    public static bool class_is_valuetype(
+        [NativeCallbackType("MonoClass*")] IntPtr klass)
+    {
+        Type t = klass.TypeFromHandleIntPtr();
+        return t?.IsValueType??false;
```

Comment:
Should there be a case where `t` is null?

Reply:
good catch, this is a relic from how I converted my MonoClass to Type, before Alex mentioned the TypeFromHandleIntPtr function. will remove the check from related PRs too